### ISSUE TITLE
Some Drug names can return false positive from Twitter

### ISSUE THREAD

**Issue by BTMurphy** [42sixsolutions/claire#165]
For example, Lyrica is the name of a R&B musician.

**Comment by ghochard42six** [comment]
Yeah, we saw this pretty early on.  Apparently a lot of bands name themselves after drugs.  So "Drug A Sucks!" ... are they talking about the drug, or the band?
